languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -20889,6 +20889,32 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             </rule>
         </rulegroup>
 
+        <rulegroup id="NO_ACCENT_ON_VERBAL_E" name="Falta de acento agudo sobre 'é'." default="temp_off">
```

Comment:
There's a similar existing rule that could perhaps be edited CONFUSÃO_E_É. This is part of issue [5564](https://github.com/languagetooler-gmbh/languagetool-premium/issues/5564). If you find more confusion pairs, you can check this issue and/or expand it.

Reply:
Moved it to the already existing rule at d57ece80cf